src/components/playlist-item/playlist-item.scss

```diff
@@ -51,7 +51,7 @@ $playlist-item-metadata-height: 46px;
       background-size: cover;
       max-height: 100%;
       img {
-        max-width: fit-content; // override style from kms
+        max-width: initial; // override style from kms
```

Comment:
why was it fit-content ?

Reply:
without any value then the videos on the list are squeezed, when put any value on chrome it looks ok, but safari needs this specific value to look good also